test/com/google/enterprise/adaptor/sharepoint/SharePointAdaptorTest.java

```diff
@@ -44,6 +44,8 @@
 import com.google.enterprise.adaptor.sharepoint.SharePointAdaptor.SharePointUrl;
 import com.google.enterprise.adaptor.sharepoint.SharePointAdaptor.SiteUserIdMappingCallable;
 import com.google.enterprise.adaptor.sharepoint.SharePointAdaptor.SoapFactory;
+import com.google.enterprise.adaptor.testing.RecordingDocIdPusher;
+import com.google.enterprise.adaptor.testing.UnsupportedDocIdPusher;
```

Comment:
UnsupportedDocIdPusher is unused.

Reply:
Replaced local copy of UnsupportedDocIdPusher with library UnsupportedDocIdPusher. 
DocIdPusher object is the instance of UnsupportedDocIdPusher.